robot/rover/systemd/ros-rover-start.service

```diff
@@ -4,11 +4,11 @@ After=ip-emailer.service
 Wants=ip-emailer.service
 
 [Service]
-#User=odroid
-# For some unkown reason this service stopped working when running as user odroid
+#User=nvidia
+# For some unkown reason this service stopped working when running as user nvidia
 User=root
 Type=simple
-ExecStart=/home/odroid/Programming/robotics-prototype/robot/util/rosRoverStart/runRosRoverStart.sh
+ExecStart=/home/nvidia/Programming/robotics-prototype/robot/util/rosRoverStart/runRosRoverStart.sh
```

Comment:
Shouldn't you be using $USER

Reply:
Won't work cause the systemd service has to run under root for some reason

Reply:
Thing is, you can change the service to run as non-root but then certain processes later in the pipeline will fail and need to be worked around. Some more digging can be done as to where specifically it messes up, but seeing as we're not going to be switching OBC super often I'd say it's fine to leave this as it is for now. Maybe we can open an issue to investigate this for after this PR is merged.

Reply:
I can try testing this out again for some investigation :eyes: :wine_glass: